commcare_connect/templates/audit/audit_report_task_modal.html

```
@@ -0,0 +1,96 @@
+  <div @click.outside="open = false"
+       class="bg-white rounded-lg shadow-lg w-full max-w-3xl overflow-hidden relative">
+    <div class="metric-container px-6 py-4">
+      <h3 class="text-lg font-semibold text-white">
+        {% blocktrans with name=entry.opportunity_access.user.name %}{{ name }}: Weekly Performance Report{% endblocktrans %}
+      </h3>
+      <p class="text-sm text-gray-100">
+        {{ report.period_start }} – {{ report.period_end }}
+      </p>
+    </div>
+
+    <form method="post"
```

HTMLHint (1.9.2)

[error] Special characters must be escaped : [ < ].

(spec-char-escape)